core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -292,22 +289,6 @@ class Analyzer(
       sender ! VoidResponse
   }
 
-  def handleReloadFiles(files: List[SourceFileInfo]): RpcResponse = {
-    val (existing, missingFiles) = files.partition(_.exists())
```

Comment:
is there a reason why we're throwing away the existence check? I recall this was causing problems